### PR TITLE
Fetch document on reorder to enable publish button

### PR DESCRIPTION
**Pull request by jcontonio**

### CODE REVIEW

File: web/src/client/js/actions/field.js [hunk 1]
```diff
@@ -54,11 +54,12 @@ export const updateFieldOrder = (projectId, documentId, fieldId, newOrder) => {
...
-      // If something bad happens, just fetch the whole document
-      dispatch(fetchDocument(documentId))
-      return
-    }
+    dispatch(fetchDocument(documentId))
+    // if (globalErrorCodes.includes(status)) {
+    //   // If something bad happens, just fetch the whole document
+    //   dispatch(fetchDocument(documentId))
+    //   return
+    // }
```
Comment (dheniges) [inline web/src/client/js/actions/field.js]: What is this changing/fixing? If I uncomment this, I still see the document updated in the client and can hit the publish button.

**Review by dheniges — COMMENTED** [review]
Not sure what's going on here that commenting out error handling changes?

**Comment by jcontonio** [comment]
Updated

**Review by dheniges — APPROVED** [review]
:+1:

**Pull request merged**